.github/workflows/check.yml

```diff
@@ -60,12 +60,33 @@ jobs:
           curl -fsSL https://cli.moonbitlang.com/install/unix.sh | bash
           echo "$HOME/.moon/bin" >> $GITHUB_PATH
 
+      - name: upgrade-core
```

Comment:
Do we really needs this though, it can be automatically solved after 1 day? (bump core)

Reply:
@bzy-debug

Reply:
don't really need